front/lib/actions/mcp_internal_actions/constants.ts

```diff
@@ -1060,7 +1060,7 @@ The directive should be used to display a clickable version of the agent name in
     isPreview: true,
     tools_stakes: {
       text_to_speech: "low",
-      text_to_discussion: "low",
+      text_to_dialogue: "low",
```

Comment:
it's `text_to_dialogue` in MCP declaration: https://github.com/dust-tt/dust/blob/f27cf206339ea7e9e08c7a3d03cbd4e92996f025/front/lib/actions/mcp_internal_actions/servers/speech_generator.ts#L122